engineyard/ey-cookbooks-stable-v5
# CC-1113 Removed 3DES and DES-CBC3-SHA from cipher list

Description of your patch
-------------

Remove DES and 3DES from the default cipher list

Recommended Release Notes
-------------

Remove DES and 3DES from the default cipher list

Estimated risk
-------------

Low

Components involved
-------------

$ git diff --name-only next-release
cookbooks/nginx/templates/default/default_ssl_cipher.erb

Description of testing done
-------------

* Apply cookbooks w/ fix on ssl enabled environment
* Verify the cipher list matches the cookbook change
* Verify chef completes without issue

QA Instructions
-------------

Same as testing above.
